pages/index.js

```diff
@@ -3,63 +3,117 @@ import styles from '../styles/Home.module.css'
 
 export default function Home() {
   return (
-    <div className={styles.container}>
+    <div>
       <Head>
-        <title>Create Next App</title>
+        <title>Alpha Site</title>
         <link rel="icon" href="/favicon.ico" />
       </Head>
 
-      <main className={styles.main}>
-        <h1 className={styles.title}>
-          Welcome to <a href="https://nextjs.org">Next.js!</a>
-        </h1>
+      <header>
+        <figure>
+          <img src="#" alt="logo.jpg"></img>
+          <figcaption>
+            Governement of Canada
+            Gouvernement du Canada
+          </figcaption>
+        </figure>
 
-        <p className={styles.description}>
-          Get started by editing{' '}
-          <code className={styles.code}>pages/index.js</code>
-        </p>
+        <section>
```

Comment:
Let's do `<div>` here. Usually you want to use `<section>` in the page body, not the header element.